source/strings.tex

```diff
@@ -3617,7 +3617,7 @@
 \effects
 Equivalent to:
 \begin{codeblock}
-return basic_string_view<charT, traits>(this.data(), pos1, n1).compare(sv);
```

Comment:
Almost all of them appear to be very recent additions and redundant. I would prefer to remove all of these occurrences that are neither necessary ([filebuf.assign]/1's `this->` is necessary since `close` is a member of a dependent base class) nor significantly improve clarity.